blog/_posts/2018_10_02_akaringular_final_blog.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Summer Down with Over 200 Commits and Over 70 Issues Closed 💻 🎊
+layout: post
+created_at: Wed Sep 05 2018
+permalink: blog/2018-09-05-team-akaringular
```

Comment:
Hi @StlMaris123 ! Lovely blog post. Thanks for the PR. Just a minor comment here: Permalink and file name should be the same. So, please change them. 

I don't have any other comments for improving your blog post but if you wish to add some stats about your work, feel free to send revised commit. I plan to publish your post sometime at the end of this week so you can send revised commit until then. Thanks.